OpenRA.Mods.Common/Widgets/CheckboxWidget.cs

```diff
@@ -23,12 +23,14 @@ public class CheckboxWidget : ButtonWidget
 		public Func<bool> IsChecked = () => false;
 		public int CheckOffset = 2;
 		public bool HasPressedState = ChromeMetrics.Get<bool>("CheckboxPressedState");
+		public new Color TextColorDisabled = ChromeMetrics.Get<Color>("TextDisabledColor");
```

Comment:
Why don't we just rely on `TextColorDisabled` and `GetColorDisabled` already defined on the base `ButtonWidget` class?

In particular `ButtonWidget` defines:
```c#
public Color TextColor = ChromeMetrics.Get<Color>("ButtonTextColor");
public Color TextColorDisabled = ChromeMetrics.Get<Color>("ButtonTextColorDisabled");
```

So that means instead of using the `ButtonTextColor`/`ButtonTextColorDisabled` pair of styles which presumably work well together we are using `ButtonTextColor`/`TextDisabledColor` which might not necessarily work well?

Reply:
@RoosterDragon you guessed correctly. Checkboxes inherit from buttons but they don't look like buttons - the text on a checkbox is ultimately a label and not a button and should get its colors from the same place that labels do. I was looking for a better way to do this - thanks for your suggestion, I will look into it. Or if you have a better idea, please share :-)

Reply:
Updated the checkbox widget to use the approach suggested here.